app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -1324,6 +1324,7 @@ class BrowserFragment : WebFragment(), LifecycleObserver, View.OnClickListener,
                 swipeRefresh!!.isRefreshing = false
 
                 updateSecurityIcon(session)
+                bringURLTLDIntoView()
```

Comment:
Does this work well with URLs that do not contain any domains, e.g. URLs with IP addresses?

Also: Since we are doing that on the main thread. Is that matcher efficient enough to not block unnecssarily?